ibexa/cdp/4.2/config/packages/ibexa_cdp.yaml

```diff
@@ -0,0 +1,17 @@
+#ibexa:
+#    system:
+#        default:
+#            cdp:
+#                # Account number can be obtained from User Account settings in Ibexa CDP dashbaord
```

Comment:
```suggestion
#                # Account number can be obtained from User Account settings in Ibexa CDP dashboard
```

Reply:
And it is not User Account settings but Accounts/Company accounts settings, you cannot find it under your profile settings.